.github/workflows/ci.yml

```diff
@@ -298,4 +298,4 @@ jobs:
       run: |
         set PYTHONPATH=%PYTHONPATH%;%GITHUB_WORKSPACE%/bin/%KRATOS_BUILD_TYPE%
         set PATH=%PATH%;%GITHUB_WORKSPACE%/bin/%KRATOS_BUILD_TYPE%/libs
-        python kratos/python_scripts/run_tests.py -l small -c python
+        python kratos/python_scripts/run_tests.py -l nightly -c python
```

Comment:
This is what I wanted to know! thx @philbucher

Reply:
it is a hack, but unfortunately as of now there is no better way to do this :/
I hope to soon have time to improve it!